row_012_DB/row_012_DB_unsafe.py

```
@@ -0,0 +1,417 @@
+
+    connection.commit()
+    connection.close()
+
+
+def get_first_group_stations(user_id: int) -> []:
+    connection = sql.connect('userbase.db')
+    cursor = connection.cursor()
+
+    cursor.execute(f"""
+    SELECT question_id, question_opened FROM users_stations where user_id = '{user_id}' and question_group = 0
+    order by question_opened;""")
```

Ruff (0.14.1)

Possible SQL injection vector through string-based query construction

(S608)